composer.json

```diff
@@ -1,30 +1,27 @@
 {
-    "name": "shopgate/connect-integration-sdk",
+    "name": "shopgate/connect-integration-sdk-php",
     "description": "A PHP framework supporting the integration with the Shogate Connect Platform.",
     "homepage": "https://github.com/shopgate/connect-integration-sdk",
     "license": "Apache-2.0",
     "version": "1.0.0-alpha",
-    "config": {
-        "platform": {
-            "php": "5.3.3"
-        }
-    },
     "authors": [
         {
             "name": "Shopgate",
             "email": "interfaces@shopgate.com"
         }
     ],
     "require": {
-        "php": ">=5.3",
+        "php": ">=5.4",
```

Comment:
thought we were dumpin' everything below 5.6?

Reply:
We just dumped the tests, so we don't have to use the "precise" dist from travis anymore.
Techincally, we just dropped 5.3 because of the short array annotation